### ISSUE TITLE
Blank constants report missing mapping

### ISSUE THREAD

**Issue by johnhearn** [mapstruct/mapstruct#622]
Hi,
When mapping a constant to a target string property, if the constant is an empty string such as:
```java
@Mapping(constant = "", target = "string") 
```
then a the mapping is ignored and an error *Missing target property: "string"* is reported. If the constant is changed to any non-blank value (for example, to a space character) the warning disappears.
This is not the behaviour I was expecting since an empty string is being explicitly set to the target property. I have been able to set a blank string using the workaround:

```java
@Mapping(expression = "java(\"\")", target = "string")
```
though this is obviously less pleasing.

**Comment by agudian** [comment]
Once #618 is fixed, we should be able to solve this quite nicely.

**Comment by johnhearn** [comment]
Sorry, just noticed that this is a duplicate of #588

**Comment by agudian** [comment]
Right - closing it :)

Thanks anyway for the report! :+1:

**Comment by gunnarmorling** [comment]
Merged the fix for #618 to master.
